Doc/whatsnew/3.10.rst

```diff
@@ -596,6 +602,24 @@ Changes in the Python API
   a 16-bit unsigned integer.
   (Contributed by Erlend E. Aasland in :issue:`42393`.)
 
+* The ``loop`` parameter to nearly all :mod:`asyncio` functions has been removed
+  following deprecation in Python 3.8. For example, a coroutine that currently
+  look like this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep(loop=loop)
+
+  Can *usually* be replaced with this::
+
+     import asyncio
+
+     async def foo(loop):
```

Comment:
```suggestion
     async def foo():
```

While it's not essential to remove it from foo(), it makes more sense to remove it since it no longer would serve a purpose after usage of *loop* is removed.